packages/expo-video/ios/NowPlayingManager.swift

```diff
@@ -182,9 +182,7 @@ class NowPlayingManager: VideoPlayerObserverDelegate {
     }
 
     return await withCheckedContinuation { continuation in
-      fetchMetadataQueue.async {
-        continuation.resume(returning: mediaItem.asset.metadata)
-      }
+      return continuation.resume(returning: mediaItem.asset.metadata)
```

Comment:
You no longer need to use a continuation, returning `mediaItem.asset.metadata` should just work